components/ContinentSvg/Antarctica/Antarctica.jsx

```diff
@@ -0,0 +1,30 @@
+import Svg, { G, Path } from "react-native-svg";
```

Comment:
using a library like 'react-native-icomoon' you might have an easier time using svgs. That can read from a single icon json file with the svg information and properly pass it through into an icon. A lot of ways to skin a cat here. If you want to continue using react-native-svg I'd recommend making this <Svg/> piece a bit more reusable so you can just pass a path in and it will output any svg.

Reply:
In the case of these SVGs, not only the paths differ. Some of the props passed to react-native-svg's Svg and G components also differ, namely the viewBox and transform props.  I could pass these in to a component that also accepts the paths, but they seem to be kind of intrinsic to the original SVGs that I converted.... I think. Also, some of these svgs are composed of many Paths. For example, Asia.jsx has 89 paths in it. I'll need more time to research icomoon.